cgi-bin/LJ/User/Message.pm

```diff
@@ -75,7 +75,15 @@ sub email_raw {
 sub has_same_email_as {
     my ( $u, $other ) = @_;
     croak "invalid user object passed" unless LJ::isu( $u ) && LJ::isu( $other );
-    return lc( $u->email_raw ) eq lc( $other->email_raw );
+
+    my $email_1 = lc( $u->email_raw );
```

Comment:
For next time, the parenthesis are unnecessary in this usage and stylistically to be avoided.